test/import_export_regex_test.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 Workiva Inc.
```

Comment:
2018

Reply:
This is present in all the files; I'll make a separate PR following this to update them if that's cool.